openreview/journal/process/eic_revision_process.py

```diff
@@ -22,8 +22,8 @@ def process(client, edit, invitation):
         note=openreview.api.Note(id=submission.id,
             content= {
                 '_bibtex': {
-                    'value': journal.get_bibtex(submission, journal.accepted_venue_id, certifications=submission.content.get('certifications', {}).get('value'))
+                    'value': journal.get_bibtex(submission, journal.accepted_venue_id, certifications=submission.content.get('certifications', {}).get('value'), [])
```

Comment:
Shouldn't it be like this?

```suggestion
                    'value': journal.get_bibtex(submission, journal.accepted_venue_id, certifications=submission.content.get('certifications', {}).get('value',[]))
```